pootle_pytest/fixtures/models/permission.py

```diff
@@ -20,7 +20,6 @@ def pootle_content_type(db):
         'model': 'directory',
     }
     content_type, created = ContentType.objects.get_or_create(**args)
-    content_type.name = 'pootle'
```

Comment:
Just wondering why this change is not on the "Django 1.8: ContentType name field removed" commit which provides a very detailed explanation.